### PR TITLE
created amp extension for playspace player including an example page …

### PR DESCRIPTION
**Pull request by rares-mihai-preda**
…and added it to the build-system/compile/bundles.config.js

### CODE REVIEW

File: examples/connatix-ps-player.amp.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  <script async custom-element="amp-connatix-ps-player" src="https://cdn.ampproject.org/v0/amp-connatix-ps-player-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+  <h2>Connatix AMP examples</h2>
+  <h3>Connatix Player Responsive</h3>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
```
Comment (mariusszabo) [inline examples/connatix-ps-player.amp.html]: Example pages need to look proper :)) remove filler stuff

File: examples/connatix-ps-player.amp.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <h3>Connatix Player Fixed</h3>
```
Comment (mariusszabo) [inline examples/connatix-ps-player.amp.html]: We don't support fixed right

File: examples/connatix-ps-player.amp.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  <amp-connatix-ps-player 
+    data-player-id="cb489cc6-770b-40cd-ac64-afa18e43acb6"
+    layout="fixed"
+    width="640"
+    height="432">
+  </amp-connatix-ps-player>
+
+  <h3>Connatix Player Responsive - With Media Id</h3>
```
Comment (mariusszabo) [inline examples/connatix-ps-player.amp.html]: With Story Id*

File: extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+import {Services} from '../../../src/services';
+import {addParamsToUrl} from '../../../src/url';
+import {dict} from '../../../src/utils/object';
+import {getData} from '../../../src/event-helper';
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {removeElement} from '../../../src/dom';
+import {userAssert} from '../../../src/log';
+
+export class AmpConnatixPsPlayer extends AMP.BaseElement {
```
Comment (mariusszabo) [inline extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js]: this class has a naming convention I think, AmpConnatixPlayspacePlayer or something

File: extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+    super(element);
+
+    /** @private {string} */
+    this.playerId_ = '';
+
+    /** @private {string} */
+    this.storyId_ = '';
+
+    /** @private {string} */
+    this.iframeDomain_ = 'https://test.amp.connatix.com';
```
Comment (mariusszabo) [inline extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js]: iframeDomain should be the legit one

File: extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    Services.preconnectFor(this.win).url(
+      this.getAmpDoc(),
+      this.iframeDomain_,
+      onLayout
+    );
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return isLayoutSizeDefined(layout);
```
Comment (mariusszabo) [inline extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js]: here , if we don't support fixed layout, I think we should return different

File: extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+    // applyFillContent so that frame covers the entire component.
+    this.applyFillContent(iframe, /* replacedContent */ true);
+
+    element.appendChild(iframe);
+    this.iframe_ = /** @type {HTMLIFrameElement} */ (iframe);
+
+    // Return a load promise for the frame so the runtime knows when the
+    // component is ready.
+    const a = this.loadPromise(iframe);
+    // this.attemptChangeSize(432, 640).then(
```
Comment (mariusszabo) [inline extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js]: remove commented code

File: extensions/amp-connatix-ps-player/OWNERS [hunk 8]
```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'rarespreda'}, {name: 'ampproject/wg-ui-and-a11y'}],
```
Comment (mariusszabo) [inline extensions/amp-connatix-ps-player/OWNERS]: ampproject/wg-ui-and-a11y ?

**Review by mariusszabo — CHANGES_REQUESTED** [review]

File: extensions/amp-connatix-player/0.1/amp-connatix-player.js [hunk 9]
```diff
@@ -36,6 +36,9 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
     /** @private {string} */
```
Comment (mariusszabo) [inline extensions/amp-connatix-player/0.1/amp-connatix-player.js]: separate everything you change in a different extension into a different PR maybe? Let's discuss

File: examples/connatix-ps-player.amp.html (hunk 1 above)
Comment (rares-mihai-preda) [reply examples/connatix-ps-player.amp.html]: I leave them for testing floating off. I'll remove them and I'll add them back after js api setting is done.

File: extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js (hunk 4 above)
Comment (rares-mihai-preda) [reply extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js]: We haven't known the name yet. We still need the product team response. From what I know, Ionut already asked them,

File: extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js (hunk 5 above)
Comment (rares-mihai-preda) [reply extensions/amp-connatix-ps-player/0.1/amp-connatix-ps-player.js]: Embed pages are now deployed on plugins and it is up-to-date now.

File: extensions/amp-connatix-ps-player/OWNERS (hunk 8 above)
Comment (rares-mihai-preda) [reply extensions/amp-connatix-ps-player/OWNERS]: same as it is in extensions/amp-connatix-player/OWNERS

File: examples/connatix-story-player.amp.html [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+  <script async custom-element="amp-connatix-story-player" src="https://cdn.ampproject.org/v0/amp-connatix-story-player-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+  <h2>Connatix Playspace Player AMP examples</h2>
+
+  <h3>Connatix Player Responsive - Landscape</h3>
+  <amp-connatix-story-player
+    data-player-id="3a8ea6ad-d705-4dd1-8bf0-9b0edec53598"
+    data-orientation="1"
```
Comment (mariusszabo) [inline examples/connatix-story-player.amp.html]: use strings portrait/landscape instead of 0-1

**Review by mariusszabo — CHANGES_REQUESTED** [review]

File: extensions/amp-connatix-story-player/0.1/amp-connatix-story-player.js [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+     // Orientation is mandatory
+     // 0 - Portrait
+     // 1 - landscape
+     this.orientation_ = userAssert(
+      element.getAttribute('data-orientation'),
+      'The data-orientation attribute is required for <amp-connatix-story-player> %s',
+      element
+    );
+
+    const orientation = parseInt(this.orientation_, 10);
```
Comment (mariusszabo) [inline extensions/amp-connatix-story-player/0.1/amp-connatix-story-player.js]: use this logic in userAssert

**Review by mariusszabo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**